src/components/FormFieldDropdown.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CreatableSelect from 'react-select/creatable';
+import elementStyles from '../styles/isomer-cms/Elements.module.scss';
+
+const calculateBoolean = (booleanVal) => {
+    if (booleanVal && booleanVal !== 'false') {
+        return 'true'
+    }
+    return 'false'
+}
+
+const creatableSelectHandler = (callback, dropdownEvent, id) => {
+    callback({
+        target: {
+            id,
+            value: dropdownEvent.value,
+        },
+    })
+}
+
+const FormFieldDropdown = ({
+    title,
+    defaultValue,
+    value,
+    id,
+    errorMessage,
+    onFieldChange,
+    isRequired,
+    style,
+}) => (
+    <>
+      <div className={elementStyles.formDropdown}>
+        <div className={`d-flex align-items-center ${elementStyles.formDropdownLabel}`}>
+            <span>{`${title}:`}</span>
+        </div>
+        <CreatableSelect
+            isClearable
```

Comment:
We should remove the `isClearable` parameter - clearing the option causes an error currently

Reply:
Resolved with [bc17015](https://github.com/isomerpages/isomercms-frontend/pull/275/commits/bc1701505ed202660f3e6e472dd03d66507858fb)